instrumentation/sidekiq/Appraisals

```diff
@@ -1,5 +1,9 @@
 # frozen_string_literal: true
 
+appraise 'sidekiq-8.0' do
+	gem 'sidekiq', '~> 8.0'
+end
```

Comment:
@dmathieu - When we need to skip an appraisal based on a Ruby version, we usually add a condition with a `Gem::Version` comparision: 

```suggestion
if Gem::Version.new(RUBY_VERSION) >= Gem::Version.new('3.2.0')
  appraise 'sidekiq-8.0' do
    gem 'sidekiq', '~> 8.0'
  end
end
```